servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublisherFlatMapSingle.java

```diff
@@ -390,12 +399,12 @@ public void onSuccess(@Nullable R result) {
 
             @Override
             public void onError(Throwable t) {
-                if (!source.delayError) {
+                if (source.maxDelayedErrors <= 0) {
```

Comment:
Values < 0 are not allowed, consider using `parent.source.maxDelayedErrors == 0` to make it more clear what `0` value means.